### PR TITLE
fix(curriculum): validating café menu html tag

### LINKED ISSUE #46539: HTML validation errors and warnings for Cafe menu practice project
Related to https://github.com/freeCodeCamp/freeCodeCamp/issues/46463#issuecomment-1155180861
The final code of the project has been validated with three different html validators (https://www.freeformatter.com/html-validator.html, https://appdevtools.com/html-validator, https://validator.w3.org/), resulting in these errors and warnings:

~~- [ ] Warning: Article lacks heading. Consider using “h2”-“h6” elements to add identifying headings to all articles. For all 3 validators~~
- [ ] Error: Stray start tag “html”. For all 3 validators
```
From line 62, column 1 to line 62, column 6.
</body> <html>
```
- [ ] Warning: Consider adding a “lang” attribute to the “html” start tag to declare the language of this document. For all 3 validators

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #46539

image

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Looks Great! Good Job!!!

**Comment by RandellDawson** [comment]
@ojeytonwilliams So you were never able to figure out a way to make it so tests could be written like the following:
```
assert(document.querySelector('html').getAttribute('lang') === 'end');
);
```
instead of having to use regex like:
```
assert(code.match(/<html\s*lang=('|")en\1\s*>/i));
```

@Sboonny I don't think you want to put `\s*` between `html` and `lang` in the regular expression unless you want to accept `<htmllang="en">` as valid html.

File: curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f33071498eb2472b87ddee4.md [hunk 1]
```diff
@@ -19,10 +19,10 @@ You should have the `DOCTYPE` declaration.
 assert(code.match(/<!DOCTYPE html>/i));
 ```
 
-You should have an opening `<html>` tag.
+You should have an opening `<html>` tag, and set `lang` attribute to `en`.
```
Comment (moT01) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f33071498eb2472b87ddee4.md:22]:
```suggestion
You should have an opening `<html>` tag with a `lang` attribute of `en`.
```
Same

**Review by moT01 — COMMENTED** [review]
I left a few minor suggestions to try and clarify the wording a little. Other than that, this LGTM 🎉

Looks like this attribute doesn't get added to the DOM so I think we need to use regex on this one for now.
Screen Shot 2022-06-26 at 1 11 35 PM

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Review by Sembauke — APPROVED** [review]

**Pull request merged**